src/clj/witan/send/model/data_products.clj

```diff
@@ -187,29 +156,28 @@
   (let [by-state (future (summarise-results
                           (partial roll-up-calendar-year-by-state
                                    (states/calculate-valid-states-from-setting-academic-years valid-states))
-                          simulations))
+                          simulations)) ;; FIXME - this should be a state filter for all data products
```

Comment:
What does this mean?

Reply:
I think we probably want this filter everywhere

Reply:
sorry still don't follow?